apps/feedback/pkg/llmclient/llm_client.go

```diff
@@ -153,7 +153,6 @@ func (c *LLMClient) PromptForShortIssueTitle(ctx context.Context, userText strin
 	var content bytes.Buffer
 	if err := t.Execute(&content, userText); err != nil {
 		return "", fmt.Errorf("executing template: %w", err)
-
```

Comment:
Just a tiny tiny nit, I am making changes in this file, is it possible to undo this change so it doesn't conflict? 🙏